From 1d1c0514158abb66388ee4eb44764d201203a863 Mon Sep 17 00:00:00 2001
From: Jakob Bornecrantz <jakob@vmware.com>
Date: Mon, 19 Apr 2010 15:29:41 +0100
Subject: [PATCH] Only match against event[0-9] in udev rules

It looks like the udev rule was matched against two devices which caused
the driver to be loaded twice. We shouldn't match against the other
device because it will very seldom send bogus events. Some of the bug
this caused was buttons getting stuck.
---
 tools/69-xorg-vmmouse.rules.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/69-xorg-vmmouse.rules.in b/tools/69-xorg-vmmouse.rules.in
index 0d7f028..602afb6 100644
--- a/tools/69-xorg-vmmouse.rules.in
+++ b/tools/69-xorg-vmmouse.rules.in
@@ -1 +1 @@
-ACTION=="add|change", ENV{ID_INPUT_MOUSE}=="?*", ATTRS{description}=="i8042 AUX port", PROGRAM="__BIN_PREFIX__/vmmouse_detect", ENV{ID_INPUT.tags}="vmmouse"
+ACTION=="add|change", ENV{ID_INPUT_MOUSE}=="?*", ATTRS{description}=="i8042 AUX port", KERNEL=="event[0-9]*", PROGRAM="__BIN_PREFIX__/vmmouse_detect", ENV{ID_INPUT.tags}="vmmouse"
-- 
1.7.0.1

